src/current/v23.2/set-up-physical-cluster-replication.md

```diff
@@ -0,0 +1,388 @@
+    {{site.data.alerts.end}}
+
+1. Add your cluster organization and {{ site.data.products.enterprise }} license to the cluster:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SET CLUSTER SETTING cluster.organization = 'your organization';
+    ~~~
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SET CLUSTER SETTING enterprise.license = 'your enterprise license';
+    ~~~
+
+1. Confirm the status of your virtual cluster: {% comment %}Link to SQL ref page here{% endcomment %}
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SHOW VIRTUAL CLUSTERS;
+    ~~~
+
+    The output will include the `system` interface and the `application` virtual cluster:
+
+    ~~~
+    id |    name     | data_state | service_mode
+    ---+-------------+------------+---------------
+    1  | system      | ready      | shared
+    2  | template    | ready      | none
+    3  | application | ready      | shared
+    (3 rows)
+    ~~~
+
+    Because this is the primary cluster rather than the standby cluster, `data_state` of all rows is `ready`, rather than `replicating` or another status.{% comment %}link to SQL reference or monitoring page here{% endcomment %}
+
+### Create a replication user and password
+
+The standby cluster connects to the primary cluster's system interface using an identity with the `REPLICATION` privilege. Connect to the primary cluster's system interface and create a user with a password:
+
+1. From the primary's system interface SQL shell, create a user and password:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE USER {your username} WITH PASSWORD '{your password}';
+    ~~~
+
+1. Grant the [`REPLICATION` system privilege]({% link {{ page.version.version }}/security-reference/authorization.md %}#supported-privileges) to your user: {% comment %}Link to detail on replication system privilege{% endcomment %}
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    GRANT SYSTEM REPLICATION TO {your username};
+    ~~~
+
+    If you need to change the password later, refer to [`ALTER USER`]({% link {{ page.version.version }}/alter-user.md %}).
+
+### Connect to the primary virtual cluster (optional)
+
+1. If you would like to run a sample workload on the primary's application virtual cluster, open a new terminal window and use [`cockroach workload`]({% link {{ page.version.version }}/cockroach-workload.md %}) to run the workload.
+
+    For example, to initiate the [`movr`]({% link {{ page.version.version }}/movr.md %}) workload:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cockroach workload init movr "postgresql://root@{node IP or hostname}:{26257}/?options=-ccluster=application&sslmode=verify-full&sslrootcert=certs/ca.crt&sslcert=certs/client.root.crt&sslkey=certs/client.root.key"
+    ~~~
+
```

Comment:
```suggestion
    cockroach workload init movr "postgresql://root@{node_advertise_address}:{node_advertise_port}/?options=-ccluster=application&sslmode=verify-full&sslrootcert=certs/ca.crt&sslcert=certs/client.root.crt&sslkey=certs/client.root.key"
    ~~~

Replace `{node_advertise_address}` and `{node_advertise_port}` with a node's `--advertise-address` IP address or hostname and port.
```
WDYT? I think including a static port might cause some confusion, but I don't feel super strongly about it.

Reply:
Added and linked to the `cockroach start` flags section for `advertise-addr`